### PR TITLE
Add Disclaimer that parse-kv does not work in DCR transformations

### PR DESCRIPTION
**Pull request by JustinGrote**
From https://github.com/Azure/Azure-Sentinel/issues/9792

### CODE REVIEW

**Comment by Jak-MS** [comment]
@shsagir 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: data-explorer/kusto/query/parse-kv-operator.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ The following extraction modes are supported:
 * [**Non-specified delimeter**](#nonspecified-delimiter): Extraction with no need to specify delimiters. Any nonalphanumeric character is considered a delimiter.
 * [**Regex**](#regex): Extraction based on [regular expressions](regex.md).
 
+This operator is not supported in [DCR transformations](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/data-collection-transformations). Consider regex or parse instead.
```
Comment (mjosephym) [inline data-explorer/kusto/query/parse-kv-operator.md:21]:
```suggestion
:::moniker range="monitor, sentinel"
>[!NOTE]
> The `parse-kv` operator isn't supported in [DCR transformations](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/data-collection-transformations). Use [**`regex`**](#regex) or the [parse operator](parse-operator.md) instead.
::: moniker-end
```

File: data-explorer/kusto/query/parse-kv-operator.md [hunk 2]
```diff
@@ -18,6 +18,11 @@ The following extraction modes are supported:
 * [**Non-specified delimeter**](#nonspecified-delimiter): Extraction with no need to specify delimiters. Any nonalphanumeric character is considered a delimiter.
 * [**Regex**](#regex): Extraction based on [regular expressions](regex.md).
 
+:::moniker range="monitor, sentinel"
```
Comment (mjosephym) [inline data-explorer/kusto/query/parse-kv-operator.md:21]:
```suggestion
::: moniker range="azure-monitor || microsoft-sentinel"
```

**Comment by guywi-ms** [comment]
@JustinGrote Thank you very much for your dedication to our documentation! 

Providing clear information about which KQL operators are supported in various Azure Monitor scenarios (such as transformations, search jobs, and cross-service queries) is crucial for our users. We're currently exploring options to add this information automatically, ensuring that the content remains complete and up-to-date for all KQL operators.

For now, we will close this PR, but please know that your contribution is highly valued.

Thank you!

#please-close

**Comment by JustinGrote** [comment]
@guywi-ms thank you, can you back up your statement by creating a specific issue, rollup issue, or draft PR to address this concern? I'm worried this will just go off into the ether.

**Pull request closed without merging**